src/applications/vaos/api/index.js

```diff
@@ -273,21 +273,29 @@ export function checkPastVisits(
 export function getRequestLimits(facilityId, typeOfCareId) {
   let promise;
   if (USE_MOCK_DATA) {
-    promise = Promise.resolve({
-      data: {
-        attributes: {
-          requestLimit: 1,
-          numberOfRequests: facilityId.includes('984') ? 1 : 0,
-        },
-      },
+    promise = new Promise(resolve => {
+      setTimeout(() => {
+        resolve({
+          data: {
+            id: facilityId,
+            attributes: {
+              requestLimit: 1,
+              numberOfRequests: facilityId.includes('984') ? 1 : 0,
+            },
+          },
+        });
+      }, 1000);
     });
   } else {
     promise = vaosApiRequest(
       `/v0/facilities/${facilityId}/limits?type_of_care_id=${typeOfCareId}`,
     );
   }
 
-  return promise.then(resp => resp.data.attributes);
+  return promise.then(resp => ({
+    ...resp.data.attributes,
+    facilityId: resp.data.id,
```

Comment:
I'd like to keep this as `id` and avoid customizing the response structure beyond attributes flattening in the api file. I think that makes the functions in here more predictable.